2019/python/lar_constraints.py

```diff
@@ -994,9 +994,10 @@ def v696_const(self, row):
 			row["aus_code_16"] = ""
 
 		#number of reported systems must match the number of reported results
-		result_enums = ("1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14","15", "16")
+		result_enums = ("1111","1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14","15", "16", 
+			"17", "18", "19", "20", "21", "22", "23", "24", "")
 		for i in range(1, len(aus_sys)):
-			if aus_sys[i] in ("1", "2", "3", "4", "5") and aus_results[i] not in result_enums:
+			if aus_sys[i] in ("1", "2", "3", "4", "5", "6", "") and aus_results[i] not in result_enums:
```

Comment:
This change looks like it would make the blank acceptable for AUS 1 as well as making it non-repeatable for AUS 2-5. 

adding blank with a condition that AUS sys is not AUS 1 and then adding the blank to the enum set inside that condition should fix it.

Reply:
Thanks, David. I was hoping to get your thoughts on this. I am going to push up further changes.

Reply:
I think if you take the blanks out of both AUS and AUS results value sets it should work.

Reply:
Hi David - I think that changes were not reflected by my last commit.